doc/modules/linear_model.rst

```diff
@@ -100,12 +99,19 @@ of squares:
 
 .. math::
 
-   \min_{w} || X w - y||_2^2 + \alpha ||w||_2^2
+   \min_{w} || X w + w_0 1_s - y||_2^2 + \alpha ||w||_2^2
```

Comment:
```suggestion
   \min_{w} || X w + w_0 - y||_2^2 + \alpha ||w||_2^2
```
The OLS equation above just uses $|| X w + w_0 - y||_2^2$. Why should we start here with introducing $1_s$?

Reply:
To make the shapes of all terms consistent (since w_0 is a scalar). I think I multiply terms by `1_s` somewhere in the derivation later to get some terms to simplify.